docs/integrations/ai-engines/neuralforecast.mdx

```diff
@@ -1,8 +1,190 @@
 ---
-title: NeuralForecast
+title: Nixtla's NeuralForecast Integration with MindsDB
 sidebarTitle: NeuralForecast
 ---
 
-<Note>
-This page is a work in progress.
-</Note>
+Nixtla’s NeuralForecast provides a diverse array of neural forecasting models, prioritizing their ease of use and resilience. These models encompass a spectrum of options, including traditional networks like MLP and RNNs, as well as cutting-edge innovations such as NBEATS, NHITS, TFT, and various other architectural approaches.
+
+You can learn more about its features [here](https://nixtla.github.io/neuralforecast/).
+
+## How to bring NeuralForecast Models to MindsDB
+
+Before creating a model, you will need to create an ML engine for NeuralForecast using the `CREATE ML_ENGINE` statement:
+
+```sql
+CREATE ML_ENGINE neuralforecast
+FROM neuralforecast;
+```
+
+Once the ML engine is created, we use the `CREATE MODEL` statement to create the NeuralForecast model in MindsDB.
+
+```sql
+CREATE MODEL model_name
+FROM data_source
+  (SELECT * FROM table_name)
+PREDICT column_to_be_predicted
+ORDER BY date_column
+GROUP BY column_name, column_name, ...
+WINDOW 12 -- model looks back at sets of 12 rows each
+HORIZON 3 -- model forecasts the next 3 rows
+USING ENGINE = 'neuralforecast';
+```
+
+To ensure that the model is created based on the NeuralForecast engine, include the `USING` clause at the end.
+
+## Example
+
+Let's go through an example of how to use Nixtla's NeuralForecast with MindsDB to forecast monthly expenditures based on historical data.
+
+Please note that before using the NeuralForecast engine, you should create it from the MindsDB Cloud editor with the below command:
+
+```sql
+CREATE ML_ENGINE neuralforecast
+FROM neuralforecast;
+```
+
+You can check the available engines with this command:
+
+```sql
+SHOW ML_ENGINES;
+```
+
+If you see the NeuralForecast engine on the list, you are ready to follow the tutorials.
+
+We use a table from our MySQL public demo database, so let’s start by connecting MindsDB to it:
+
+```sql
+CREATE DATABASE mysql_demo_db
+WITH ENGINE = 'mysql',
+PARAMETERS = {
+    "user": "user",
+    "password": "MindsDBUser123!",
+    "host": "db-demo-data.cwoyhfn6bzs0.us-east-1.rds.amazonaws.com",
+    "port": "3306",
+    "database": "public"
+};
+```
+
+Now that we’ve connected our database to MindsDB, let’s query the data to be used in the example:
+
+```sql
+SELECT *
+FROM mysql_demo_db.historical_expenditures
+LIMIT 3;
+```
+
+Here is the output:
+
+```sql
++------------+----------+-------------+
+| month      | category | expenditure |
++------------+----------+-------------+
+| 1982-04-01 | food     | 1162.6      |
+| 1982-05-01 | food     | 1150.9      |
```

Comment:
Maybe manually change the category to have one more category except food just as an example

Reply:
ok so now statsforecast uses food category, neuralforecast - clothing, and timegpt - industry.
so the examples differ a bit.